enhancements/update/to-multi-arch.md

```diff
@@ -0,0 +1,167 @@
+
+#### Standalone Clusters
+The proposal targets standalone clusters.
+
+#### Single-node Deployments or MicroShift
+The proposal should apply to SNO as CVO treats it the same as a standalone cluster.
+The version of MicroShift is not managed by [dnf](https://docs.redhat.com/fr/documentation/red_hat_build_of_microshift/4.17/html-single/installing_with_an_rpm_package/index#microshift-install-rpm), not CVO and thus MicroShift does not hit the issue described here.
+
+### Implementation Details/Notes/Constraints
+
+The changes in this section are also reflected in [clusteroperator.md](../../dev-guide/cluster-version-operator/dev/clusteroperator.md).
+
+Each operator creates a pull request that includes:
+
+- in the `ClusterVersion` manifest:
+
+  - a new element with `name=operator-image` and `version=placeholder.url.oc.will.replace.this.org/placeholdernamespace:<operator-tag-name>` where `<operator-tag-name>` is the name of the tag for the operator in [image-references](./operators.md#how-do-i-ensure-the-right-images-get-used-by-my-manifests).
+  - OPTIONALLY a new element with `name=operand-image` and `version=placeholder.url.oc.will.replace.this.org/placeholdernamespace:<operand-tag-name>` where `<operand-tag-name>` is the name of the tag for an operator's operand in [image-references](./operators.md#how-do-i-ensure-the-right-images-get-used-by-my-manifests).
+
+
+- in the `deployment` manifest, environment variables or flags with a placeholder for an image pull specification that will be replaced when the payload is built.
+
+- in the operator's implementation, the operator takes the values of the environment variables or the flags and use them to populate the `clusteroperatror.status.versions`.
+
+A special operator: [machine-config-operator](https://github.com/openshift/machine-config-operator) (MCO).
+It is special because it is the last operator to upgrade in a cluster upgrade and it usually takes longest for MCO to upgrade. With the new populated elements in `clusteroperator.status.versions`, the ending point of migrating to multi-arch will become accurate which improves the overall UX the most.
+The pull request [MCO#4637](https://github.com/openshift/machine-config-operator/pull/4637) is created for the purpose.
+Other operators may take the pull request as an example when it is ready to report the image pull specifications in the operator's `.status.versions`.
+
+### Risks and Mitigations
+
+As we allow operators to modify their implementation on its own pace. The upgrade progress of migrating to multi-arch only gets better when more operators take the actions. No risks are foreseen at the moment.
+
+### Drawbacks
+
+All OpenShift cluster operators have to modify their implementation to make the whole migration process accurate.
+
+## Open Questions [optional]
+
+## Test Plan
```

Comment:
Do we want to cover the rescue path in case migration hit a failure?
For example, what is the next move if 4.x.y-amd64 -> 4.x.y-multi cannot be completed?
This might not be directly relevant to this enhancement though.

Reply:
yeah I'd say that's outside of scope of this enhancement

Reply:
Included in the no-goal section.